src/_product-updates/2024-08-10-feedbacks-gathering.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Documentation feedback 
```

Comment:
I would suggest:
- Feedback button
- Collect feedback
- Start collecting feedback
- Get feedback from users

Reply:
I think "Get feedback" from users works well. So everybody know what we're talking about